plugins/LilyPad/Tooltips.cpp

```diff
@@ -63,8 +63,13 @@ LPWSTR dialog_message(int ID, bool *updateText)
                    L"Double-click a device in the list or right-click it and select \"Test Device\" to display a continuously updated list of the state of all inputs on the selected device.\n"
                    L"Use this option to check if all the inputs on a controller function properly.\n\n"
                    L"Right-click and select \"Refresh\" to update the list of devices in case a recently connected device has not shown up yet.";
+        case IDC_G_DI:
+            return L"(Legacy) Enable this if your gamepad doesn't support Xinput.\n"
+                   L"Disable for DualShock 4 (PS4 controllers) and probably others.";
         case IDC_G_XI:
-            return L"Xbox 360/Xbox One controllers (or devices/software imitating such) only";
+            return L"For Xbox 360/ Xbox One controllers (or devices supporting Xinput).\n\n"
+                   L"Running through Steam as a non-Steam game might be required for the controllers to work properly.\n\n"
```

Comment:
> "Running through Steam as a non-Steam game might be required for the controllers to work properly"

I would specify that this is for controllers which do not support Xinput. For such controllers Steam ends up converting the controller's inputs into Xinput, allowing you to use Xinput bindings in Lilypad. Steam is completely unnecessary for controllers which already use Xinput, and this makes it sound like Steam is still sometimes needed for Xinput controllers.

Reply:
Well i tried to be open with it:
Enable this if you have Xinput, if it doesnt work then do it via steam.